applications/trilinos_application/custom_python/trilinos_pointer_wrapper.h

```diff
@@ -0,0 +1,59 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					     Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+#if !defined(KRATOS_TRILINOS_POINTER_WRAPPER_H_INCLUDED)
+#define KRATOS_TRILINOS_POINTER_WRAPPER_H_INCLUDED
+
+// System includes
+
+// External includes
+
+//Trilinos includes
+#include "Epetra_Vector.h"
+#include "Epetra_FEVector.h"
+
+// Project includes
+#include "trilinos_application.h"
+#include "trilinos_space.h"
+
+namespace Kratos
+{
+
+typedef TrilinosSpace<Epetra_FECrsMatrix, Epetra_FEVector> TrilinosSparseSpaceType;
```

Comment:
Can you add the comment why this class is needed?
Basically what you wrote in the first post of the PR